api-conversion/README.md

```diff
@@ -0,0 +1,128 @@
+# API Conversion CLI Tool
+
+With Strimzi's CRD evolving, there needs to be a way to easily migrate existing CR configuration to the new CRD schema.
+That is why we introduced the `v1beta2` version of the schema which removes several deprecated fields.
+To make it as easy as possible to convert the existing custom resources, we provide this conversion tool.
+
+The tool can operate in two modes:
+* Converting YAML files
+* Converting Kubernetes resources
+
+You can list the available features using the `help` command:
+
+```
+> bin/api-conversion.sh help
+Usage: bin/api-conversion.sh [-hV] [COMMAND]
+Conversion tool for Strimzi Custom Resources
+  -h, --help      Show this help message and exit.
+  -V, --version   Print version information and exit.
+Commands:
+  help                  Displays help information about the specified command
+  convert-file, cf      Convert Custom Resources from YAML file
+  convert-resource, cr  Convert Custom Resources directly in Kubernetes
+```
+
+## Converting YAML files with custom resources
+
+One way to use the tool is to convert YAML files. 
+The converted YAML files can be applied to your Kubernetes cluster either directly or for example using GitOps mechanism.
+The input YAML file is specified using the `--file` option.
+
+The converted YAML will be by default printed to standard output.
+You can use the additional options `--in-place` to write the converted YAML into the original source file.
+You can also specify another output file using the `--output` option.
+
+Following example commands show how the tool can be used:
+
+```
+# Convert input.yaml and print the converted resource to standard output
+bin/api-conversion.sh convert-file --file input.yaml
+
+# Read resources from input.yaml, convert it, and store it in input.yaml
+bin/api-conversion.sh convert-file --file input.yaml --in-place
+
+# Read resources from input.yaml, convert it, and store it in output.yaml
+bin/api-conversion.sh convert-file --file input.yaml --output output.yaml
+```
+
+The convertor supports multi-document YAMLs.
+When your input YAML contains multiple Strimzi custom resources, they will be all converted.
+If the input YAL also contains any other Kubernetes resources, they will be kept in the output file without being modified.
+
+You can also list the help for the file conversion:
+
+```
+> bin/api-conversion.sh help convert-file
+Usage: bin/api-conversion.sh convert-file [-d] -f=<inputFile> [-ll=<level>]
+       [-o=<outputFile> | [--in-place]]
+Convert Custom Resources from YAML file
+  -d, --debug              Use debug
```

Comment:
```suggestion
  -d, --debug              Runs the tool in debug mode
```

Reply:
Is this "debug mode" as in "attach a debugger" or debug logging, or something else?